src/main/java/ch/njol/skript/effects/EffSuppressWarnings.java

```diff
@@ -42,11 +42,11 @@ public class EffSuppressWarnings extends Effect {
 
 	static {
 		Skript.registerEffect(EffSuppressWarnings.class,
-			"[local[ly]] suppress [the] (1:conflict|2:variable save|3:[missing] conjunction[s]|4:starting [with] expression[s]) warning[s]"
+			"[local[ly]] suppress [the] (1:conflict|2:variable save|3:[missing] conjunction[s]|4:starting [with] expression[s]|5:local variable type[s]) warning[s]"
 		);
 	}
 
-	private static final int CONFLICT = 1, INSTANCE = 2, CONJUNCTION = 3, START_EXPR = 4;
+	private static final int CONFLICT = 1, INSTANCE = 2, CONJUNCTION = 3, START_EXPR = 4, LOCAL_TYPES = 5;
```

Comment:
Yeah we could just use a switch statement to assign the ScriptWarning value based on the parse mark

Reply:
Or some ordinal like:
```java
int mark = parseResult.mark;
if (mark < 0 || mark > Enum.values().length)
	return false;
warning = Enum.values()[mark];

Reply:
Oh yeah this is already done in init. Could probably store the ScriptWarning we are suppressing and use that for the toString switch instead of the int constants

Reply:
I was thinking of making it it's own type. This is user accessable, so making it a ClassInfo is just simplier for us to add new enums in the future.

I wasn't going to address this here, but I guess I can.

Reply:
We could do that too. You don't have to do it for this PR though